reflect_test.go

```diff
@@ -176,3 +176,18 @@ func TestBaselineUnmarshal(t *testing.T) {
 
 	require.Equal(t, strings.Replace(string(expectedJSON), `\/`, "/", -1), string(actualJSON))
 }
+
+func TestMinimumValue(t *testing.T) {
```

Comment:
This test shouldn't exist anymore - use the table driven tests above.

Reply:
fixed